srcpkgs/auto-cpufreq/files/README.voidlinux

```diff
@@ -0,0 +1,4 @@
+If you are using GNOME, you should turn off the GNOME Power Profiles daemon.
```

Comment:
KDE also uses it

Reply:
![image](https://github.com/void-linux/void-packages/assets/84625092/473d9236-8f3a-4505-a2bf-758e22254796)

Reply:
```
xbps-query -RX power-profiles-daemon
```

lists `gnome` as the only reverse dependency. Is `power-profiles-daemon` an optional dependency in KDE?

I have mentioned only GNOME in the README because that's what upstream warns about.

Reply:
specifically `powerdevil` that requires `power-profile-daemon`, idk why it is not explicitly added as runtime dependency

Reply:
Arch package: https://archlinux.org/packages/extra/x86_64/powerdevil/
adds it as optional dependency

Reply:
Ok, done.